Re-enable test-e2e/SubGroup/load_store.cpp

martygrant opened the pull request:
Fixes #14765

dm-vodopyanov commented:
@martygrant please

* always add `pi-port` tag to **issues** related to this activity
* follow https://github.com/intel/llvm/blob/sycl/sycl/doc/developer/ContributeToDPCPP.md#commit-message
* provide meaningful name for the pull request
* provide description for the pull request - especially, why the test was re-enabled

martygrant commented:
@dm-vodopyanov thanks! I've added the tag, updated the commit message and added a description to the PR. I think the title of the PR is plenty descriptive enough?

martygrant commented:
@intel/llvm-gatekeepers this is ready to merge
